code/modules/spells/spell_types/shapeshift.dm

```diff
@@ -14,6 +14,7 @@
 	var/revert_on_death = TRUE
 	var/die_with_shapeshifted_form = TRUE
 	var/convert_damage = TRUE //If you want to convert the caster's health to the shift, and vice versa.
+	var/convert_blood = FALSE //If you want to convert the caster's blood level to the shift, and vice versa.
```

Comment:
this should probably fall under convert damage, i'm not really sure what situation warrants where you would want 1 but not the other.

Reply:
I left it as separate vars in case we wanted something like a vampire who could combat shape in the future.

Reply:
then you'd set both to FALSE, no?

I agree sometimes you might want the vampire to combat shape, I just don't know why you'd want it to combat shape out of blood but not damage (or vice versa)

Reply:
Alright, I'll put the change under convert_damage instead of its own var.

Reply:
Done.